omod/src/main/java/org/openmrs/module/spa/servlet/SpaServlet.java

```diff
@@ -30,13 +31,20 @@
 	protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {
 		RequestDispatcher dispatcher = req.getRequestDispatcher("/module/spa/master-single-page-application.form");
 
-		req.setAttribute("openmrsBaseUrlContext", new String(req.getContextPath()));
+		String openmrsBaseUrlContext = req.getContextPath();
+		AdministrationService service = Context.getAdministrationService();
+		String importMapUrl = service.getGlobalProperty(GP_IMPORT_MAP_URL);
```

Comment:
Minor point, but you could use the getGlobalProperty(GP, default) here to avoid this if/then, see line 41/42 that sets the spa base url context.  Would also be good to pull the "/frontend/import-map.json" out into a constant like DEFAULT_SPA_BASE_URL

Reply:
Good point, thanks!